<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_salesmanager</class>
 <widget class="QWidget" name="add_salesmanager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>销售经理</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>500</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_money">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>销售额：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_percentage">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>300</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>提成比例：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_salary">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>400</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>月薪：</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_name">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>95</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_money">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>195</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_percentage">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>295</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_salary">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>395</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
